File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int a = 0;
        int count = 0;
        for(int i=0; i<nums.size(); i++)
        {
            if(nums[i] == 1)
            {
                a++;
                count = max(a, count);
            }
            else
                a = 0;
        }
        return count;
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int a = 0, count = 0;
        for(int i=0; i<nums.length; i++)
        {
            if(nums[i] == 1)
            {
                a++;
                count = Math.max(count, a);
            }
            else
                a = 0;
        }
        return count;
    }
}